### PR TITLE
Demo added

### PR DESCRIPTION
**Pull request by nicolas-camacho**
Demo app

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: sample-app/index.html [hunk 1]
```diff
@@ -0,0 +1,286 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (ferazoguerrero) [inline sample-app/index.html:2]: could you add a screenshot or video of the web running?

Comment (nicolas-camacho) [reply sample-app/index.html:2]: Screenshot 2024-07-25 at 4 40 21 AM
Screenshot 2024-07-25 at 4 45 21 AM

**Review by SergioFierro — APPROVED** [review]

**Review by ferazoguerrero — APPROVED** [review]

**Comment by kmorope** [comment]
Nice to have:
- Add step-by-step instructions on how to run the sample app on the readme

**Comment by nicolas-camacho** [comment]
> Nice to have:
> 
> * Add step-by-step instructions on how to run the sample app on the readme

This command was added as part of the readme in the release PR, you could check it here:
https://github.com/twilio/twilio-verify-passkeys-web/pull/8

**Review by kmorope — APPROVED** [review]

**Pull request merged**